app/src/main/res/values/strings.xml

```diff
@@ -339,9 +339,10 @@
   </plurals>
   <string name="undo">Undo</string>
 
-  <plurals name="recent_pages">
+  <string name="recent_pages">Recent pages</string>
+  <plurals name="recent_pages" translatable="false">
```

Comment:
can you either remove `translatable="false"` or remove the plurals from `values-ar` (should this be okay in locales that require plurals of zero, one, two, ...?)

also, super minor nitpick - can we name the plural different from the string?